packages/block-editor/src/components/index.native.js

```diff
@@ -91,6 +91,7 @@ export { default as DefaultBlockAppender } from './default-block-appender';
 export { default as __unstableEditorStyles } from './editor-styles';
 export { default as Inserter } from './inserter';
 export { default as InserterButton } from './inserter-button';
+export { default as ImageLinkDestinationsScreen } from './image-link-destinations';
```

Comment:
Given this component is only used within the `block-editor` packages, we should like consider it private and _not_ export it. WDYT? 

Note: in making it private, we would need to import it via a relative path.